crates/sui-cluster-test/src/cluster.rs

```diff
@@ -199,13 +199,18 @@ impl Cluster for LocalNewCluster {
         // This cluster has fullnode handle, safe to unwrap
         let fullnode_url = test_cluster.fullnode_handle.rpc_url.clone();
 
+        let migrated_methods = if options.use_indexer_experimental_methods {
```

Comment:
@666lcz CI & benchmark will need an extra flag to run experimental methods, do you have a pointer of that? thanks!